prompts/sources.go

```diff
@@ -120,14 +120,38 @@ func sourceBaseForm(quickstart *Quickstart) (*QuickstartState, error) {
 		fileLocation = *quickstart.Defaults.SchemaPath
 	}
 
-	if _, err := charm_internal.NewForm(huh.NewForm(
-		getBaseSourcePrompts(quickstart.WorkflowFile, &sourceName, &fileLocation, &authHeader, &authSecret)...),
-		"Let's setup a new source for your workflow.",
-		"A source is a compiled set of OpenAPI specs and overlays that are used as the input for a SDK generation.").
-		ExecuteForm(); err != nil {
+	useSampleSpec := false
+	_, err := charm_internal.NewForm(huh.NewForm(
+		huh.NewGroup(
+			huh.NewSelect[bool]().
+				Title("Do you have an existing OpenAPI spec?").
+				Description("You can provide a local file path or a remote file URL to your OpenAPI spec.").
+				Options(
+					huh.NewOption("Yes", false),
+					huh.NewOption("No, use a sample OpenAPI spec", true),
+				).
+				Value(&useSampleSpec),
+		),
+	)).ExecuteForm()
+	if err != nil {
 		return nil, err
 	}
 
+	if useSampleSpec {
+		quickstart.IsUsingSampleOpenAPISpec = true
+		// This is a placeholder to pass document validation, it will be overwritten when we know where the output directory is
+		fileLocation = "https://example.com/placeholder"
```

Comment:
Wait why are we doing this. The sample spec is a local file

Reply:
We should just be skipping the source configuration prompts entirely

Reply:
As explained by the comment we don't know the file location yet as we will write it only once we know the outDir. If we don't put something here the formatDocument() call will fail. Open to suggestions

Reply:
Thanks for the suggestion

Reply:
I thought about making the placeholder a const string and referencing the const string inside QuickStart.go would that be more obvious to you?

Reply:
Won't we add to the workflow file when we set the sample spec